### DISCUSSION TITLE
[R] Autocomplete "function" and "return"

### DISCUSSION THREAD

**Discussion by mns-nordicals** [posit-dev/positron#4839] [Q&A] [answered]
I recently installed Positron and I didn't add any exstentions. 

However I find the autocompletion a bit strange. Two very common function: "function()" and "return()" don't show up in the autocomple as shown in the picture below. 

Does anyone else experience this or is it just me ? 

![image](https://github.com/user-attachments/assets/149ca209-744e-4798-b1bc-c1e1c0e9a726)

![image](https://github.com/user-attachments/assets/1ccd12d1-f510-4484-87b0-8cf290064b17)

**Comment by juliasilge** [comment, marked as answer]
That is fairly surprising! :open_mouth: Thank you for this report; I've opened an issue at https://github.com/posit-dev/positron/issues/4842.

As a workaround for both of these, we do have [snippets](https://code.visualstudio.com/docs/editor/userdefinedsnippets) available. Try out typing the first three characters, `fun` and `ret`, to use the snippets.

**Reply by mns-nordicals** [reply]
Thnak you for confirming and opening an issue.
